1-exercises/A-accessing-values/exercise3.js

```diff
@@ -20,7 +20,9 @@ let basketballTeam = {
     - console.logs the name of each player on a new line
 */
 
-// write code here
+basketballTeam.topPlayers.sort().forEach((player) => {
```

Comment:
Nice use of method chaining. 
If You want to know more just check: 
https://dev.to/isiakaabd/method-chaining-in-javascript-154a
https://www.youtube.com/watch?v=5rwuKH-zdos